src/core/a-animation.js

```diff
@@ -138,7 +139,11 @@ module.exports.AAnimation = registerElement('a-animation', {
         }
 
         // Create Tween.
-        return new TWEEN.Tween(cloneValue(from))
+        if (from['material.color']) {
```

Comment:
I'm not sure.  I am sure it was added for a reason because of some case to lose an object reference since all object assignments are references in JS.  I haven't dug in to see if this is a shallow clone or a deep clone.  

I mainly made this commit to share how I was able to 'fix' the issue, but I wasn't completely sure why these changes fixed it. I know its just an object reference somewhere.